scene/2d/camera_2d.cpp

```diff
@@ -159,8 +173,17 @@ Transform2D Camera2D::get_camera_transform() {
 			}
 		}
 
+		// TODO ..
+		// There is a bug here.
+		// Smoothing occurs rather confusingly
+		// during the call to get_camera_transform().
+		// It may be called MULTIPLE TIMES on certain frames,
+		// therefore smoothing is not currently applied only once per frame / tick,
+		// which will result in some haphazard results.
```

Comment:
Just to note, this mentioned bug isn't introduced by physics interpolation, it is pre-existing, and this note is to remind to fix it, or make obvious if it appears in a later issue.

Fixing may need some care as `Camera2D` is pretty central to 2D games and there would be risk of regressions.